backend/fms_core/services/report.py

```diff
@@ -99,22 +145,39 @@ def get_report_as_excel(report_data):
     return out_stream.getvalue()
 
 
-def list_reports():
+def list_reports() -> EnhancedName:
+    """
+    Lists all reports that are currently available.
+
+    Returns:
+        List of name and display name dictionary for each report.
+    """
     return Report.objects.values("name", "display_name")
```

Comment:
This doesn't return `EnhancedName`. It returns `ValuesQuerySet[Report, dict[str, Any]]` which can be narrowed down to `Sequence[EnhancedName]`